packages/-ember-data/tests/integration/record-data/record-data-test.ts

```diff
@@ -12,7 +12,52 @@ class Person extends Model {
   name;
 }
 
-// TODO Make the class implement the RD interface
+class House extends Model {
+  // TODO fix the typing for naked attrs
+  @attr('string', {})
+  name;
+
+  @belongsTo('person', { async: false })
+  landlord;
+
+  @hasMany('person', { async: false })
+  tenants;
+}
+
+const houseHash = {
+  type: 'house',
+  id: '1',
+  attributes: {
+    name: 'Moomin'
+  }
+};
+
+const davidHash = {
+  type: 'person',
+  id: '1',
+  attributes: {
+    name: 'David'
+  }
+};
+
+const runspiredHash = {
+  type: 'person',
+  id: '2',
+  attributes: {
+    name: 'Runspired'
+  }
+};
+
+const igorHash = {
```

Comment:
defining these outside of the context of a `beforeEach` is leaky.

Reply:
fixed